polaris-ui/src/mock-api/handlers.ts

```diff
@@ -165,9 +172,10 @@ export const setupHandlers = ({
     rest.get(makeApiPath(routes.FILE_ROUTE), (req, res, ctx) => {
       const { documentId } = req.params;
 
-      const blobName = pipelinePdfResultsDataSources[
-        sourceName
-      ]()[0].documents.find(
+      const docs = pipelinePdfResultsDataSources[sourceName]()[0]
+        .documents as DocumentWithPdfBlobName[];
+
+      const blobName = docs.find(
```

Comment:
Since we are removing blobName, cant we just use cmsOriginalFileName here in stead of blobName removing the extension part.

Reply:
I've made a note to refactor this as per your comment